collections/mapzen-dem.yaml

```diff
@@ -97,22 +97,23 @@ CubeDimensions:
     extent:
       - -180
       - 180
+    reference_system: 4326
   y:
     type: spatial
     axis: y
     extent:
       - -85
       - 85
+    reference_system: 4326
   t:
     type: temporal
     extent:
       - null
       - null
-    step: P5D
```

Comment:
This PR removes it :P

Reply:
Yeah, I was just asking what the expected value really is in that case. I can't find anywhere in the STAC spec that `step` is optional.
 fftr

Reply:
It's optional because otherwise it would be written "required" as it is for type and extend ([here](https://github.com/stac-extensions/datacube#temporal-dimension-object)). What should be the expected value in those cases is not really clear as it is a 1 time acquired only product. Anyway, resolving.